SpecializedServers/SVO/Games/PS3/Killzone2.cs

```diff
@@ -588,9 +588,10 @@ await SVOServerConfiguration.Database.GetAccountByName(acctNameREX, appId).Conti
                             }
                     }
                 }
-                catch (Exception e)
+                catch (Exception ex)
                 {
-
+                    LoggerAccessor.LogError($"[SVO] - Killzone2_SVOAsync thrown an assertion - {ex}");
```

Comment:
There is an extra Async in the log, after this is fixed I will push.

Reply:
I named it after the function name?

Reply:
Yeah, my bad, it is true now some are async. I will resolve and push.

Reply:
they must be async for awaited database calls so yeah..

Reply:
You can await in a sync method, using the Wait() function on a task object.

But it is true using awaiter ffuncs is more optimized.